crates/subspace-farmer/src/dsn.rs

```diff
@@ -170,3 +173,24 @@ where
 
     Ok(())
 }
+
+/// Starts the relaying node. Returns a Node instance along with a stop handle for the node runner.
+pub async fn configure_relay_server(listen_on: Vec<Multiaddr>) -> (Node, JoinHandle<()>) {
```

Comment:
This is fine for tests, but I don't think we want this for actual production usage. In production scenario we want to provide a complete config (so we can persist identity or else every time we start child nodes their dialable address will use random peer ID), we also want to expose `node_runner` directly, so that caller can decide how and for how long to drive it.